Cargo.lock

```diff
@@ -231,7 +231,7 @@ checksum = "965c2d33e53cb6b267e148a4cb0760bc01f4904c1cd4bb4002a085bb016d1490"
 dependencies = [
  "proc-macro2",
  "quote",
- "syn 2.0.99",
+ "syn 2.0.100",
```

Comment:
Could you revert this file please? We don't need to do `cargo update` in the same PR

Reply:
Sure thing, done in https://github.com/FuelLabs/fuel-core/pull/2947/commits/03c6d5fa61c22fe0bd0ec54007743b03a7c0edec